docker_push

```diff
@@ -5,9 +5,6 @@ docker push anniebedford/todo-app-prod:latest
 docker push anniebedford/todo-app-prod:$TRAVIS_COMMIT
 docker tag anniebedford/todo-app-prod registry.heroku.com/todo-app-ab/web
 
-
-#echo "$HEROKU_API_KEY" | docker login --username="abedford@gmail.com" --password-stdin registry.heroku.com
-echo $HEROKU_API_KEY 
 docker login --username="abedford@gmail.com" --password="$HEROKU_API_KEY" registry.heroku.com
```

Comment:
You don't need to use your email here, --username=_ will work fine and Heroku will work out your username from the API key (or you can do `heroku container:login` instead)